common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -461,13 +478,21 @@ private AuthenticationConstants() {
          * String for request id returned from Evo.
          **/
         public static final String REQUEST_ID_HEADER = "x-ms-request-id";
+
+        /** String for the broker version. */
+        public final static String ADAL_BROKER_VERSION = "x-client-brkrver";
```

Comment:
Misspelled constant

Reply:
This is a merge from the Unity, so I would like to keep the "backward compatibility"